src/libs/Navigation/AppNavigator/ReportScreenIDSetter.ts

```diff
@@ -16,6 +16,8 @@ type ReportScreenIDSetterComponentProps = {
 
     /** Whether user is a new user */
     isFirstTimeNewExpensifyUser: OnyxEntry<boolean>;
+
+    reportMetadata: OnyxCollection<ReportMetadata>;
```

Comment:
Oops missed this, but props missing a comment.
A part of checklist 🙈

Reply:
>  All props are defined accurately and each prop has a /** comment above it */

Reply:
Updated @MonilBhavsar

Reply:
Thanks!